test/e2e/common/expansion.go

```diff
@@ -645,14 +640,15 @@ var _ = framework.KubeDescribe("Variable Expansion", func() {
 
 func testPodFailSubpath(f *framework.Framework, pod *v1.Pod) {
 
-	pod, err := f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(pod)
-	Expect(err).ToNot(HaveOccurred(), "while creating pod")
+	var podClient *framework.PodClient
+	podClient = f.PodClientNS(f.Namespace.Name)
+	pod = podClient.Create(pod)
```

Comment:
simplify to `f.PodClient()`? (applies to all instances)

Reply:
You mean just this line? `podClient = f.PodClientNS(f.Namespace.Name)` to this `podClient = f.PodClient()`

Reply:
yes, PodClient() defaults to f.Namespace.Name

Reply:
Okay. Thanks. I switched it and retested/ pushed up. I did a squeeze of the commits as well to keep it clean